scripts/authz_proposal_for_validator_ops.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -e
+
+# Instructions:
+#  - to run: ./authz_proposal_for_validator_ops.sh [grant/revoke]
+#  - put grantee addresses in grantee.txt
+
+if [[ -z "$1" ]]; then
```

Comment:
shoudl we force them to supply grant/revoke rather than default to either

Reply:
updated